<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="DrawingSurface" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="widget_4" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuCapture"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Device</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
          <item>
           <widget class="QComboBox" name="comboBox_devices"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnReloadDevs">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="topMargin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Vendor:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_dev_vendor">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Model:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_dev_model">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_dev_type">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Parameters</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="tableView_device_opts">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
  </widget>
  <action name="actionStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawingSurface</class>
   <extends>QWidget</extends>
   <header>drawingsurface.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
